setup.py

```diff
@@ -9,11 +9,12 @@
 
 setup(
     name="guillotina_amqp",
-    version="5.0.21.dev0",
+    version="6.0.0",
     description="Integrate amqp into guillotina",
     long_description=README,
     install_requires=[
-        "guillotina>=5.0.0.dev0,<6",
+        "guillotina>=6.0.0<7",
```

Comment:
does it mean that we no longer support version 5?

Reply:
correct, it doesn't.
I created a `5.x` of the repo with the current `master`